org.hl7.fhir.r4/src/main/java/org/hl7/fhir/r4/utils/client/FHIRToolingClient.java

```diff
@@ -103,16 +105,18 @@
 	private ClientUtils utils = new ClientUtils();
 	
 	//Pass enpoint for client - URI
-  public FHIRToolingClient(String baseServiceUrl) throws URISyntaxException {
-    preferredResourceFormat = ResourceFormat.RESOURCE_XML;
-    detectProxy();
-    initialize(baseServiceUrl);
-  }
-  
-  public FHIRToolingClient(String baseServiceUrl, String username, String password) throws URISyntaxException {
+
+	public FHIRToolingClient(String baseServiceUrl) throws URISyntaxException {
+		preferredResourceFormat = ResourceFormat.RESOURCE_XML;
+		detectProxy();
+		initialize(baseServiceUrl);
+	  }
+	
+  public FHIRToolingClient(String baseServiceUrl, String accessToken, String username, String password) throws URISyntaxException {
```

Comment:
It doesn't appear as though you ever use the username and password with the accessToken. Please create a constructor that takes in just the url and token instead of having to call the constructor over and over with null params. It makes its cleaner.

Also, add javadocs to details what is happening here.